examples/nomad-consul-separate-cluster/main.tf

```diff
@@ -72,7 +72,7 @@ data "template_file" "startup_script_nomad_server" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 module "consul_cluster" {
-  source = "git::git@github.com:gruntwork-io/terraform-google-consul.git//modules/consul-cluster?ref=v0.0.2"
+  source = "git::git@github.com:hashicorp/terraform-google-consul.git/modules/consul-cluster"
```

Comment:
It's very important that we include a `?ref=vX.Y.Z` here. Otherwise, each invocation of this module will download what is then the latest commit to the `master` branch. As of today, the latest release of https://github.com/hashicorp/terraform-google-consul is `v0.0.3`, so can you update accordingly?